update docs to reflect main git enhancements (#4980)
fishtown-analytics/docs.getdbt.com: website/docs/guides/bigquery-qs.md

    - In the command line bar at the bottom, enter `dbt run` and click **Enter**. You should see a `dbt run succeeded` message.

## Build your first model

> In the dbt Cloud IDE, you have two options for working with files:

Reviewer: ```suggestion
You have two options for working with files in the dbt Cloud IDE:
```

> - Direct editing in the protected primary branch &mdash; You can edit, format, or lint files and execute dbt commands directly in your primary git branch. Once you're ready, you can commit those changes to a new branch.

Author: ```suggestion
- Direct editing in the protected primary branch &mdash; You can edit, format, or lint files and execute dbt commands directly in your primary git branch. Since the dbt Cloud IDE prevents commits to the protected branch, you can commit those changes to a new branch.
```

> - Direct editing in the protected primary branch &mdash; You can edit, format, or lint files and execute dbt commands directly in your primary git branch. Since the dbt Cloud IDE prevents commits to the protected branch, you can commit those changes to a new branch.

Reviewer: ```suggestion
- Editing in the protected primary branch &mdash; Edit, format, or lint files and execute dbt commands directly in your primary git branch. Since the dbt Cloud IDE prevents commits to the protected branch, it prompts you to commit those changes to a new branch.
```

> - Create a new branch before editing &mdash; If you prefer to keep the primary branch unchanged, you can create a new branch before starting your edits. To do this, go to **Version Control** on the left sidebar and click **Create branch**.

Reviewer: ```suggestion
- Create a new branch before editing &mdash; If you prefer to keep the primary branch unchanged, create a new branch before starting your edits. Navigate to **Version Control** on the left sidebar and click **Create branch**.
```

> You can name the new branch `add-customers-model`.

Reviewer: ```suggestion
Name the new branch `add-customers-model`.
```
